src/main.ts

```diff
@@ -41,7 +42,9 @@ Vue.use(Toast, VUE_TOAST_GENERAL_OPTIONS)
 Vue.config.productionTip = false
 
 // todo CORE-155: inline 'store' after migration to Vue3
-const store = new Vuex.Store({})
+const store = new Vuex.Store({
+  plugins: [VuexUndoRedoPlugin],
```

Comment:
@object-required , If I'm not wrong this would require we do similar thing on voto5?
if YES, Is there a better way to just only do it within the community version, and not replicate the declaration on the consumer side?

Reply:
Great point, I moved plugin registration to `TreeBuilderToolbar.created()` lifecycle hook --
https://github.com/FLOIP/flow-builder/blob/d3fcefec4ca8e1bcb727aefd9753eb0c89282c89/src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue#L372-L374